docs/designers-developers/developers/slotfills/plugin-document-setting-panel.md

```diff
@@ -0,0 +1,26 @@
+# PluginDocumentSettingPanel
+
+This SlotFill allows adding items below the Status & Availability panel in the Document Sidebar.
+
+## Available Props
+* __name__ `string`: A string identifying the panel.
+* __className__ `string`: An optional class name added to the sidebar body.
+* __title__ `string`: Title displayed at the top of the sidebar.
+* __icon__ `(string|Element)`: The [Dashicon](https://developer.wordpress.org/resource/dashicons/) icon slug string, or an SVG WP element.
+
+## Example
+```js
+const { registerPlugin } = wp.plugins;
+const { PluginDocumentSettingPanel } = wp.editPost;
+
+const PluginDocumentSettingPanelDemo = () => (
+	<PluginDocumentSettingPanel
+		name="custom-panel"
+		title="Custom Panel"
+		className="custom-panel"
+	>
+		Custom Panel Contents
```

Comment:
Maybe an example to edit a document setting (a meta field for instance) could be more valuable?

Reply:
Great suggestion - perhaps we should apply this feedback to the other docs for SlotFill as well.

Reply:
Yeah, that makes sense. 

Also, I feel having a "slotfills" category/section in the docs is also a bit misleadinig. I'd personally prefer extensibility options organized in a more "semantic way".

Instead of SlotFills section, we could have:

 - Block extensibilty APIs
 - Publishinig flow extensibiltiy APIs 
 - Document extensibility APIs

we could continue these adaptations separately.

Thanks for the update on this one 👍

Reply:
I wonder if there is an automated way to do that, via some kind of jsdoc tag to move slotfills into a more self-documented approach? (Just a musing!)